app/importers/reading/mega_reading_processor.rb

```diff
@@ -19,11 +19,13 @@ def initialize(educator_id, options = {})
     reset_counters!
   end
 
-  def import(file_text, options = {})
+  def process(file_text, options = {})
     reset_counters!
 
     # parse
     rows = []
+    #last row before data is the header
+    # string = (@header_rows_count > 1) ? file_text.lines[@header_rows_count-1..-1].join : file_text
```

Comment:
Let's remove the comment if we don't need it anymore.

Reply:
Should `header_rows_count` be updated to two?  In the template it looks like there's the header row and then two additional rows for educators explaining things and we'd want to skip both those additional rows.  We could rename this too if it's ambiguous what "header" here means too.

Reply:
addressed in https://github.com/studentinsights/studentinsights/pull/2544/files#diff-f01c15c5800dc5ef4b4a64b7ad07baadR23